samples/bluetooth/observer/src/observer.c

```diff
@@ -9,7 +9,7 @@
 #include <zephyr/bluetooth/bluetooth.h>
 #include <zephyr/bluetooth/hci.h>
 
-#define NAME_LEN 30
+#define NAME_LEN sizeof(CONFIG_BT_DEVICE_NAME)
```

Comment:
Can't this be reverted back to how it was? Or is this change necessary?

Reply:
Ditto, separate PR